README.md

```diff
@@ -1,24 +1,14 @@
-# README
+# Bally Blogs
+## Blogging Engine
+### author: Dylan Kearney
+There are too many blogging engines in the world today, so I will build one more. The outcome of this body of work will be a basic blogging engine that encompasses some of the key elements one would encounter in software development. It also serves to provide a practical platform to learn and ask questions about what it means to develop software at WP Engine. 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+###### *The name of the prject comes from the name of where I grew up; Ballydesmond*
 
-Things you may want to cover:
+v1.0: Devise gem added to project for later use. Global navbar added for easy routing on the website for dev use. (26/08/2020)
 
-* Ruby version
+v1.1: Added static home page. Created post model with user ID, title, context and date & time when created. Set up authentication using Devise. Set up view to input the post details. User must be authenticated to create a post. Success message displayed on post creation. (01/09/2020)
 
-* System dependencies
+v1.2: Added view for list of posts. List of posts displays author, title (links to the post itself) , truncated context, date of creation. Also added a user reference to the posts table. A user can have many posts but a post may only have one user/author. (08/09/2020)
 
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+v1.3 (coming soon) : Added edit functionality. A user can edit their blog article once logged in.
```

Comment:
Generally it's a good idea to put the most recent version first so that users don't have to scroll all the way to the bottom to see what's changed in the most recent version.

Reply:
Understandable 👍